packages/perseus-core/src/widgets/free-response/free-response-util.ts

```diff
@@ -0,0 +1,23 @@
+import type {PerseusFreeResponseWidgetOptions} from "@khanacademy/perseus-core";
+
+/**
+ * For details on the individual options, see the
+ * PerseusFreeResponseWidgetOptions type
```

Comment:
Thanks for adding this doc comment! To make it even more helpful, you could make the reference a link:

```suggestion
 * {@link PerseusFreeResponseWidgetOptions} type
```